src/components/mobileOnlyAccordion/MobileOnlyAccordionItem.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { AccordionItemProps, useMediaQuery } from '@chakra-ui/react';
+
+import AccordionItem from '../accordion/AccordionItem';
+import { breakpoints } from '../../themes';
+
+const MobileOnlyAccordionItem: FC<PropsWithChildren<AccordionItemProps>> = (
+  props
+) => {
+  const { children, ...rest } = props;
+  const [isLargerThan1024] = useMediaQuery(
+    `(min-width: ${breakpoints.md.px}px)`
+  );
+  const desktopStyle = isLargerThan1024 ? { border: 'none' } : {};
```

Comment:
I assume we can override that if needed? (As for the stacking example I showed you)

Reply:
For mobile devices yes, we can change it by passing style props. But this one is completely disabling borders on desktop.

We use borders here by default on all devices, but we do not need it on desktop (as per design).
I didn't find a way how to set styles for specific media query in theme component. Maybe you could suggest something better?

Reply:
Yeha thats fine just so we can get rid of the double ones for stacking